### PR TITLE
[Do Not Merge] Adds a minecraft ruin to lavaland

### PR DESCRIPTION
**Pull request by BobJoga**
## About The Pull Request

Adds a minecraft ruin to lavaland, with some mobs and items included. Sound effects too.

## Why It's Good For The Game

Lavaland hasn't received new ruins in a long while and i thought this would be funny as hell

## Changelog
:cl:
add: Added minecraft ruin
add: Added minecraft turfs
add: Added minecraft mobs
add: Added minecraft sword and pickaxe
soundadd: Added many minecraft sounds
/:cl:

### CODE REVIEW

File: modular_skyrat/code/modules/mining/minecraft/minecraft.dm [hunk 1]
```diff
@@ -0,0 +1,234 @@
+/obj/item/pickaxe/minecraft
+	name = "Diamond Pickaxe"
+	desc = "Aw man."
+	force = 10
+	icon = 'modular_skyrat/icons/obj/minecraft_items.dmi'
+	icon_state = "pickaxe"
+	w_class = WEIGHT_CLASS_NORMAL
+	toolspeed = 0.001 //best pickaxe bro
+
+/obj/item/claymore/minecraft
```
Comment (Trilbyspaceclone) [inline modular_skyrat/code/modules/mining/minecraft/minecraft.dm]: 50% block + what ever armor pen is a lot for a easy to get sword. Like this is almost as good a katana imo

File: modular_skyrat/code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm [hunk 2]
```diff
@@ -0,0 +1,36 @@
+/mob/living/simple_animal/hostile/asteroid/goliath/zombie
+	name = "zombie"
+	desc = "Why is he so blocky?"
```
Comment (Trilbyspaceclone) [inline modular_skyrat/code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm]: if its a guy then maybe gender = MALE it

Comment (BobJoga) [reply modular_skyrat/code/modules/mob/living/simple_animal/hostile/mining_mobs/goliath.dm]: i will yes

File: modular_skyrat/code/modules/mining/minecraft/minecraft.dm (hunk 1 above)
Comment (BobJoga) [reply modular_skyrat/code/modules/mining/minecraft/minecraft.dm]: The chaplain holy claymore is pretty much this and they get it at roundstart (only a 2 damage difference

**Comment by Owai-Seek** [comment]
I'd say to post pictures but I legitimately hate terrible crossovers.

File: modular_skyrat/code/modules/mining/minecraft/minecraft.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply modular_skyrat/code/modules/mining/minecraft/minecraft.dm]: w_class = WEIGHT_CLASS_HUGE
	slot_flags = ITEM_SLOT_BACK|ITEM_SLOT_BELT
	block_chance = 30
        18 damage ye
Note the block dosnt block ANY projectiles 

minecraft sword
	w_class = WEIGHT_CLASS_NORMAL - Fits in bags
	block_chance = 50 - blocks bullets and projectiles 
        20 damage 

Its really good

Comment (BobJoga) [reply modular_skyrat/code/modules/mining/minecraft/minecraft.dm]: Alright, now that i think about it you have a point
i'm goinna make it shitty

**Comment by BobJoga** [comment]
Marked as DNM until i get sprites that fit better the SS13 mob style. Requested by Xyel.

**Comment by BobJoga** [comment]
fuck this shit im closing this pr for now fuck you

**Pull request closed without merging**